packages/core/src/core.ts

```diff
@@ -778,6 +778,108 @@ class ThresholdKey implements ITKey {
     return { newShareStores, newShareIndex };
   }
 
+  async importTssKey(
+    params: { tag: string; importKey: BN; factorPub: Point; newTSSIndex: number },
+    serverOpts: {
+      serverEndpoints: string[];
```

Comment:
I think we can get the server details from ` const rssNodeDetails = await this._getRssNodeDetails();`
except selectedServers and authSignatures